### PR TITLE
Regression test for setting random seed for ECA compiling in AQT

### PR DESCRIPTION
**Pull request by singular-value**

### CODE REVIEW

**Comment by singular-value** [comment]
See also https://github.com/SupertechLabs/cirq-superstaq/pull/404

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -88,11 +89,26 @@ def test_aqt_compile(provider: qss.SuperstaQProvider) -> None:
 def test_aqt_compile_eca(provider: qss.SuperstaQProvider) -> None:
     circuit = qiskit.QuantumCircuit(8)
     circuit.h(4)
+    mat = scipy.linalg.fractional_matrix_power(qiskit.circuit.library.CXGate().__array__(), 0.7)
+    circuit.unitary(mat, [4, 5])  # CX**0.7
```
Comment (richrines1) [inline qiskit_superstaq/daily_integration_test.py:93]: we can use a more natural qiskit gate here, e.g.
```suggestion
    circuit.crx(0.7 * pi, 4, 5)
```
(which is locally equivalent to CX**0.7)

Comment (singular-value) [reply qiskit_superstaq/daily_integration_test.py:93]: ah nice

**Review by richrines1 — APPROVED** [review]
lgtm!

**Pull request merged**